client/blocks/upsell-nudge/index.jsx

```diff
@@ -60,7 +60,7 @@ export const UpsellNudge = ( {
 		href = addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
 	}
 
-	return <Banner { ...props } showIcon={ showIcon } className={ classes } />;
+	return <Banner { ...props } showIcon={ showIcon } className={ classes } href={ href } />;
```

Comment:
@sixhours let's please explicitly audit and pass through props we need to Banner as a follow up. The args added in https://github.com/Automattic/wp-calypso/pull/40762 change what ...props is (it means everything else not declared in args).

Reply:
Yes, I will work on this today! Thanks for handling everything while I was out. <3

Reply:
Started a PR in #41143